src/reference-manual/pedantic-mode.Rmd

```diff
@@ -0,0 +1,363 @@
+
+
+# Pedantic Mode
+
+Pedantic Mode is a compilation option built into Stanc3 that warns you about potential issues in your Stan program.
+
+For example, if you compile the following program with Pedantic Mode:
+
+    data {
+      int N;
+      real x[N];
+    }
+    parameters {
+      real sigma;
+    }
+    model {
+      real mu;
+      x ~ normal(mu, sigma);
+    }
```

Comment:
This all needs to be escaped for markdown.  So do the blocks of program output.  

The rendered HTML and pdf should be a mess for this.

Reply:
Oh my, markdown. I was checking this by rendering the html to avoid these issues, and it's rendered preformatted!

There must be an equivalence between indenting for preformatting and surrounding with ticks. Yeah we should surround this stuff with ticks.

Reply:
D'oh!  I forgot that four-space indent worked like ticks!  I should've rendered it myself.  Presumably that also goes through to the LaTeX.  I'm pretty sure we've used ticks almost everywhere else.

Reply:
Yeah, I usually indent, since Pandoc and org-markdown both produce indent. Sounds like I should use ticks for consistency?

Reply:
Yeah go ahead and backtick. Saves me from having to learn.